.github/workflows/build-test.yml

```diff
@@ -0,0 +1,26 @@
+name: Build and test
+
+on:
+  pull_request:
+
+jobs:
+  build-test:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
+          fetch-depth: 0
+
+      - name: Set up JDK 8
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.8.0
```

Comment:
is this sample still on 8?

Reply:
I missed that the travis build was on 11, I'll update.